lib/r10k/cli/deploy.rb

```diff
@@ -23,6 +23,8 @@ def self.command
 
         required :c, :config, 'Specify a configuration file'
 
+        required nil, :cachedir, 'Specify a cachedir, overriding the value in config'
```

Comment:
That seems reasonable. It wouldn't take much work to update the other deploy subcommands to honor the option as well for now if you think that would be helpful.